### ISSUE TITLE
what loss function you used for training those models

### ISSUE THREAD

**Issue by SkylerZheng** [ruotianluo/ImageCaptioning.pytorch#79]

**Comment by ruotianluo** [comment]
word level cross entropy loss.

**Comment by SkylerZheng** [comment]
Thank you very much for your quick response.


Best regards,
Jian Zheng
Ph.D. Student, Electrical Engineering
Dept. of Electrical & Computer Engineering at the State University of New
York at Binghamton
Tel: (607)953-9 <%28607%29372-3633>960
Email: jzheng65@binghamton.edu <qdong3@binghamton.edu>
